src/Tribe/Aggregator/Processes/Batch_Imports.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+class Tribe__Events__Aggregator__Processes__Batch_Imports {
+	public function hook() {
+		add_filter( 'tribe_events_aggregator_build_url', [ $this, 'build_url' ], 10, 3 );
```

Comment:
:no_entry_sign: Missing doc comment for function hook()

<sub>hash: 81130abaa1a695eb37bd846bab2a95a7</sub>